src/Events/Integrations/Plugins/Event_Tickets/Site_Health/Subsection.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Class that handles interfacing with core Site Health.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Event_Tickets\Site_Health
+ */
+
+namespace TEC\Events\Integrations\Plugins\Event_Tickets\Site_Health;
+
+use TEC\Tickets\Site_Health\Abstract_Info_Subsection;
+
+/**
+ * Class The_Events_Calendar_Fields
+ *
+ * @since   TBD
+ * @package TEC\Events\Integrations\Plugins\Event_Tickets\Site_Health
+ */
+class Subsection extends Abstract_Info_Subsection {
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function is_subsection_enabled(): bool {
+		return class_exists(
+			'Tribe__Events__Main',
+			false
+		);
+	}
```

Comment:
Is this dependent on TEC? So if TEC isn't enabled, these stats won't show?

Reply:
I just realized that this was for the TEC part of it. If TEC wasn't enabled, then this class will never run. I think we can just return true in this case.